src/Rolodex.tsx

```diff
@@ -269,9 +274,9 @@ export default function Rolodex() {
               }
               return { card, tag: undefined };
             })
-            .map(({ card, tag }) => (
+            .map(({ card, tag }, index) => (
               <Card
-                key={card.callsign}
+                key={card.callsign + index}
```

Comment:
if a card gets moved around e.g. when a user changes the sort, this will make all of them be fully rerendered, not just just reordered in the DOM. blowing away the whole list on reorder isn't really the intent of `key` and could cause performance issues with large lists

if the issue is that callsigns aren't unique, firebase should return an `id` field along with each card -- it's not in the types yet, and won't be present for the draft card, but otherwise should work for this scenario?

Reply:
If each card gets an unique id, i support using that instead